### DISCUSSION TITLE
What is the price or amount data type in Prisma (Postgres)

### DISCUSSION THREAD

**Discussion by sandbox-apps** [prisma/prisma#8622] [Q&A] [answered]
How to specify a price or amount data type in prisma?

model Item {
  price Int @default(100) ?????
}

**Comment by ryands17** [comment, marked as answer]
Which type do you want to use from Postgres' native type? We have all the types listed [here](https://www.prisma.io/docs/concepts/database-connectors/postgresql#native-type-mappings).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
